simple-examples/src/manual-trigger.ts

```diff
@@ -6,6 +6,7 @@ const hatchet = Hatchet.init();
 async function main() {
   const workflowRunId = await hatchet.admin.run_workflow("example", {});
 
+  // @ts-ignore // TODO fix this
   hatchet.listener.on(workflowRunId, async (event: StepRunEvent) => {
```

Comment:
this doesn't compile, @grutt any idea what this is supposed to look like?

Reply:
I should have just let AI do the job 😁 
<img width="610" alt="image" src="https://github.com/hatchet-dev/hatchet-typescript-quickstart/assets/5013932/461a011d-eff8-42ea-93e0-084ab5929943">

Reply:
🤖